src/portfolios/portfolios.service.ts

```diff
@@ -92,4 +94,12 @@ export class PortfoliosService {
 
     return identity.portfolios.getCustodiedPortfolios(paginationOptions);
   }
+
+  public async createdAt(did: string, portfolioId: BigNumber): Promise<EventIdentifier | null> {
+    if (portfolioId.lte(0)) {
+      throw new AppValidationError('Cannot get event details for Default Portfolio');
+    }
+    const portfolio = await this.findOne(did, portfolioId);
+    return (portfolio as NumberedPortfolio).createdAt();
```

Comment:
Why would this be null?

Reply:
I guess if its not synced yet, so probably OK to leave the service as it is.

Reply:
This is how the SDK is returning. If event details is found it gives the data otherwise returns null